### ISSUE TITLE
Titanic - Hyperparameter tuning

### ISSUE THREAD

**Issue by Aakriti28** [wncc/Hello-FOSS-ML#10]
A neural network has been trained on the dataset in [this notebook](https://github.com/wncc/Hello-FOSS-ML/blob/master/Titanic%20neural%20network%20Tensorflow.ipynb) but it has very low accuracy. Your task is to tune the hyperparameters and improve the model. Some suggestions - modifying the optimizer, adding more hidden layers to the model or changing the dimension of layers, adding dropout, regularization etc.
